### PR TITLE
 [Storage][STG74]ChangeLog

### PR DESCRIPTION
**Pull request by tasherif-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-file-share/setup.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
     author_email='ascl@microsoft.com',
     url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-file-share',
     classifiers=[
-        "Development Status :: 5 - Production/Stable",
+        "Development Status :: 4 - Beta",
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-share/setup.py:61]: great job

**Review by xiafu-msft — DISMISSED** [review]

**Comment by xiafu-msft** [comment]
let's merge stg74 branch first then merge this pr

File: sdk/storage/azure-storage-file-datalake/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Release History
-## 12.1.3 (Unreleased)
-
+## 12.2.0b1 (2020-10-02)
+**New Features**
+- Added support for recursive set/update/remove Access Control on a path and sub-paths.
+- Added support for setting an expiry on files where the file gets deleted once it expires.
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-datalake/CHANGELOG.md:5]:
```suggestion
- Added support to generate directory SAS and added support to specify additional user ids and correlation ids for user delegation SAS.
```

File: sdk/storage/azure-storage-file-datalake/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
+- Added support to generate directory SAS and added support to specify additional user ids and correlation ids for user delegation SAS.
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-datalake/CHANGELOG.md:6]: I forgot to add this in my pr, use your pr to add this for convenience haha

**Review by xiafu-msft — APPROVED** [review]

**Review by rakshith91 — APPROVED** [review]

**Pull request merged**